requirements-full.txt

```diff
@@ -245,7 +245,7 @@ pytrends==4.8.0
 pytz-deprecation-shim==0.1.0.post0; python_version >= "3.7" and python_full_version < "3.0.0" or python_version >= "3.7" and python_full_version < "3.9.7" and python_full_version >= "3.6.0" or python_full_version > "3.9.7" and python_version >= "3.6"
 pytz==2022.5; python_full_version >= "3.7.1" and python_full_version < "4.0.0" and python_version >= "3.8" and (python_version >= "3.8" and python_full_version < "3.9.7" or python_full_version > "3.9.7" and python_version >= "3.8") and python_version < "4.0"
 pyupgrade==2.38.4; python_version >= "3.7"
-pywin32==304; sys_platform == "win32" and platform_python_implementation != "PyPy" and python_version >= "3.7"
+pywin32==305; sys_platform == "win32" and platform_python_implementation != "PyPy" and python_version >= "3.7"
```

Comment:
grasping for straws, but could this be related to the min32 prompt toolkit bug?

Reply:
This is a dump of poetry lock. Changing pyproject.toml is not required if you want to play around with the pinned versions

Reply:
So that answers my above question about pyproject.  This question was because on the windows build there is the following error: `  File "prompt_toolkit\output\win32.py", line 218, in get_win32_screen_buffer_info
prompt_toolkit.output.win32.NoConsoleScreenBufferError: No Windows console found. Are you running cmd.exe?`

I see this pywin bumped.  As I said, just grasping for straws,